openassessment/xblock/submission_mixin.py

```diff
@@ -258,11 +258,11 @@ def save_submission(self, data, suffix=''):  # pylint: disable=unused-argument
                     {"saved_response": self.saved_response}
                 )
             except Exception:  # pylint: disable=broad-except
-                return {'success': False, 'msg': self._("This response could not be saved.")}
+                return {'success': False, 'msg': self._("Please contact support staff.")}
```

Comment:
```suggestion
                return {'success': False, 'msg': self._("Draft could not be saved. Please contact support staff.")}
```

Reply:
The error banner already includes "We could not save your progress" so it seemed unnecessary for it to restate that "Draft could not be saved" as below. I'm not super opinionated about this though, if you think it is important to change.

> **We could not save your progress**
> Draft could not be saved. Please contact support staff.